bin/cml-runner.js

```diff
@@ -72,116 +72,116 @@ const shutdown = async (opts) => {
     }
   };
 
-  const shutdown_tf = async () => {
-    const { tf_resource } = opts;
+  const shutdownTf = async () => {
+    const { tfResource } = opts;
 
-    if (!tf_resource) {
+    if (!tfResource) {
       console.log(`\tNo TF resource found`);
       return;
     }
 
     try {
-      await tf.destroy({ dir: tf_path });
+      await tf.destroy({ dir: tfPath });
     } catch (err) {
       console.error(`\tFailed Terraform destroy: ${err.message}`);
       error = err;
     }
   };
 
-  const destroy_terraform = async () => {
+  const destroyTerraform = async () => {
     try {
-      console.log(await tf.destroy({ dir: tf_path }));
+      console.log(await tf.destroy({ dir: tfPath }));
     } catch (err) {
       console.error(`\tFailed destroying terraform: ${err.message}`);
       error = err;
     }
   };
 
   if (cloud) {
-    await destroy_terraform();
+    await destroyTerraform();
   } else {
-    RUNNER_LAUNCHED && (await unregister_runner());
+    RUNNER_LAUNCHED && (await unregisterRunner());
 
     console.log(
       `\tDestroy scheduled: ${RUNNER_DESTROY_DELAY} seconds remaining.`
     );
     await sleep(RUNNER_DESTROY_DELAY);
 
-    DOCKER_MACHINE && (await shutdown_docker_machine());
-    await shutdown_tf();
+    DOCKER_MACHINE && (await shutdownDockerMachine());
+    await shutdownTf();
   }
 
   process.exit(error ? 1 : 0);
 };
 
-const run_cloud = async (opts) => {
-  const { cloud_ssh_private_visible } = opts;
+const runCloud = async (opts) => {
+  const { cloud_ssh_private_visible: cloudSshPrivateVisible } = opts;
 
-  const run_terraform = async (opts) => {
+  const runTerraform = async (opts) => {
     console.log('Terraform apply...');
 
     const { token, repo, driver } = cml;
     const {
       labels,
-      idle_timeout,
+      idle_timeout: idleTimeout,
       name,
       single,
       cloud,
       cloud_region: region,
       cloud_type: type,
       cloud_gpu: gpu,
-      cloud_hdd_size: hdd_size,
-      cloud_ssh_private: ssh_private,
+      cloud_hdd_size: hddSize,
+      cloud_ssh_private: sshPrivate,
       cloud_spot: spot,
-      cloud_spot_price: spot_price,
-      cloud_startup_script: startup_script,
-      tf_file,
+      cloud_spot_price: spotPrice,
+      cloud_startup_script: startupScript,
+      tfFile,
       workdir
     } = opts;
```

Comment:
I think that `opts` come in snake case just because of [`decamelize-keys`](https://www.npmjs.com/package/decamelize-keys) 🤔

Reply:
:facepalm: